CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## \[future\] Polars v0.13
 
-### \[future\] py-polars 0.7.0
+### \[future\] polars 0.7.0
```

Comment:
We need to ad an extra banner now to differentiate between the Rust and the Python Polars version.

Reply:
What do you mean by banner? Maybe especially if py-polars is split out (see below), it can have its separate CHANGELOG?

Reply:
Yes, a separate CHANGELOG would be more clear indeed. Let's do that.

Reply:
#436